bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -274,6 +274,9 @@ def consume_next(self):
 
             self._validate_chunk(chunk)
 
+            if ("ReadRowsIterator" in self._response_iterator.__class__.__name__):
+                self._response_iterator.set_start_key(chunk.row_key)
```

Comment:
I think this can only happen once the row has been fully processed, so after `_save_current_row`?